src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Utility/GetPackageReferenceUtility.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NuGet.Frameworks;
+using NuGet.LibraryModel;
+using NuGet.Packaging.Core;
+using NuGet.ProjectModel;
+using NuGet.Versioning;
+
+namespace NuGet.PackageManagement.VisualStudio.Utility
```

Comment:
This GetPackageReferenceUtility.cs change comes from Martin's [PR](https://github.com/NuGet/NuGet.Client/pull/3539). He already had PR for reference type unit tests and I'm using some of his code only for reference type project unit test.